pelias/README.md

```diff
@@ -0,0 +1,168 @@
+# Setup guide for Pelias geocoding service.
+
+This guide explains how we set up the geocoding service for https://best.osoc.be.
+
+This folder contains configuration files to download/prepare/build a complete Pelias installation for Belgium
+using the BOSA open address database and various other open data sources.
```

Comment:
Add link to BOSA open address website!

Reply:
Woops forgot this!

Reply:
Fixed with 6c4a8a8a30191cb6954ca10a749311e4dfac8e49